docs/tutorial.md

```diff
@@ -0,0 +1,231 @@
+---
+layout: page
+title: Tutorial
+permalink: /tutorial/
+---
+
+In this tutorial, we'll build an example data lake and data pipeline based on airline traffic data published [here](http://stat-computing.org/dataexpo/2009/the-data.html). You can already download and unpack those files. One year is more than enough. You can also quickly grab the data from our [repo](https://github.com/datamindedbe/lighthouse/tree/master/lighthouse-demo/src/main/resources/data).
+
+Steps:
+* Do not remove this line (it will not be displayed)
+{:toc}
+ 
+### Install Lighthouse
+Start by adding the Lighthouse dependency to your project. In case you are using maven:
+```xml
+<dependency>
+    <groupId>be.dataminded</groupId>
+    <artifactId>lighthouse_2.11</artifactId>
+    <version>0.2.0</version>
+</dependency>
+```
+or sbt:
+
+```scala
+libraryDependencies += "be.dataminded" % "lighthouse_2.11" % "0.2.0"
```

Comment:
use "be.dataminded" %% "lighthouse" % "0.2.0" As spark only supports scala 2.11 atm this should be fine. We should add different versions of lighthouse once spark supports multiple scala versions